Base/res/js/Spreadsheet/runtime.js

```diff
@@ -381,7 +383,7 @@ function numericReduce(op, accumulator, cells) {
 }
 
 function numericResolve(cells) {
-    return resolve(cells).map(str => (str === "" || str == null ? NaN : integer(str)));
+    return resolve(cells).map(integer);
```

Comment:
Wouldn't the Number constructor work just fine here?

Reply:
no, new Number(null) !== NaN but parseInt(null) === NaN so that's what I used :)